### PR TITLE
Fix symbolic call of `logsumexp` with int axis.

### PR DESCRIPTION
**Pull request by hertschuh**
Using `keras.ops.math.logsumexp` with an int for `axis` in a functional model would throw an error.

### CODE REVIEW

**Review by mattdangerw — COMMENTED** [review]
Thank you!

**Review by mattdangerw — APPROVED** [review]
